rtil/src/native/character.rs

```diff
@@ -82,6 +83,11 @@ impl AMyCharacter {
     pub fn set_rotation(&mut self, pitch: f32, yaw: f32, roll: f32) {
         self.controller_mut().rotation = FRotator { pitch, yaw, roll };
     }
+    pub fn show_hud() {
+        if !AMYHUD.is_none() {
+            unsafe { (*(*AMYHUD.get() as *mut AMyHud)).b_showhud.set_bit(1, true); }
+        }
+    }
```

Comment:
This function should be in `AMyHud`. Then you also don't need to make `AMYHUD` public (which it shouldn't be).